Sources/SwiftCliCore/StitchLibrary.swift

```diff
@@ -16,3 +20,10 @@ let allowedStitchesInfo = [StitchInfo(name: "k1", incDecValue: 0, symbol: " "),
 let nonStitchAllowedInput = ["\n"]
 
 let allowedUserInput = allowedStitchesInfo.map { $0.name } + nonStitchAllowedInput
+
+func stitchLookup(stitch: String) throws -> StitchInfo {
```

Comment:
I like this solution and how it successfully encapsulates the error handling. Well done!